search-app/src/cmr/search/services/parameters/converters/spatial.clj

```diff
@@ -1,26 +1,41 @@
 (ns cmr.search.services.parameters.converters.spatial
   "Contains parameter converters for spatial parameters"
-  (:require [cmr.common-app.services.search.params :as p]
-            [cmr.search.models.query :as qm]
-            [cmr.common-app.services.search.group-query-conditions :as gc]
-            [cmr.spatial.codec :as spatial-codec]
-            [cmr.common.services.errors :as errors]
-            [clojure.string :as str]))
+  (:require
+   [clojure.string :as string]
+   [cmr.common-app.services.search.group-query-conditions :as gc]
+   [cmr.common-app.services.search.params :as p]
+   [cmr.common.config :as cfg :refer [defconfig]]
+   [cmr.common.services.errors :as errors]
+   [cmr.search.models.query :as qm]
+   [cmr.spatial.circle :as spatial-circle]
+   [cmr.spatial.codec :as spatial-codec]))
+
+(defconfig points-on-circle
+  "The number of vertices of the polygon used to approximate a circle"
+  {:default 32 :type Long})
```

Comment:
It is configurable. From the tests I have run, 32 is a magic number that provides a good balance between precision and performance. I thought about making it a search parameter, but that opens up to abuse, so I didn't do it.

Reply:
This will be somewhat coarse for large circles, and for small circles, you might run into the problem of points being too close together (the spatial library doesn't like that).

Reply:
It might be better if this were a function of the radius - bigger radius, more points.

Reply:
See my comments on the definition of minimum and maximum radius.